### ISSUE TITLE
[zelda´s version] [retro-go] [LCD-Game-Emulator][START and SELECT as extra buttons]

### ISSUE THREAD

**Issue by xhottam** [kbeckmann/game-and-watch-retro-go#179]
is it possibe use START and SELECT button as extra one instead of mapped to GAME and TIME buttons respectively?¿.  
It could be used to correctly emulate those games that have 4 actions

Thanks in advance.

**Comment by xelapo** [comment]
Yes, it would be really useful, in Mickey Mouse (Wide Screen) for example.
